### ISSUE TITLE
There is no dataset config ? 

### ISSUE THREAD

**Issue by Enescigdem** [JuanFMontesinos/Acappella-YNet#3]
Hi,
In README. md ,
Download the code and set your dataset paths at config>dataset_paths.json is said but there is no such file in code directory or it is not mentioned about the format of that json file.
Thanks in advance

**Comment by EmreOzkose** [comment]
They are added this file to gitignore file. Hence this file is not uploaded. The reason is, I think, private path issue. You can create a file configs/

where 

```
{
    "acappella": "/path/to/data/acapella_preprocessed/splits",
    "audioset": "/path/to//data/audio"
}
```

the folder `splits` is already created by `preproc/preprocess.py`. 
the folder `audio` contains folder `eval`. folder `eval` contains folder `audio`. folder `audio` contains some audios which are acapella. These folders are download in https://research.google.com/audioset/download.html. You should download 3 file: 

eval_segments.csv
Balanced train - balanced_train_segments.csv
Unbalanced train - unbalanced_train_segments.csv

and locate to preproc. 

Then you should run this : 

`streamlit run audioset.py`

Then you should copy downloaded files to /path/to//data/audio/eval/audio.

Let me know if there is a wrong step @JuanFMontesinos .

**Comment by JuanFMontesinos** [comment]
Indeed I didn't realise the file was in gitignote and the structure should be like that. Should be fixed at 1dbe364
